index.html

```diff
@@ -122,7 +128,7 @@ <h3 class="aboutmetitles">Frameworks</h3>
             <li class="fram5">Selenium</li>
         </ul>
         </div>
-        <div class="boxaboutme">
+        <div class="boxaboutme boxaboutme3">
         <i class="iconabout"><img src="./Images/icon-skills.png" alt="icon3"/></i>
         <h3 class="aboutmetitles">Skills</h3>
         <ul class="boxlist">
```

Comment:
-I suggest you to wrap all "about me" section inside one ```<section>```

Reply:
Yes, I will do that